org-node/src/client.rs

```diff
@@ -587,6 +587,21 @@ impl Client {
                     )
                     .map_err(|e| Error::SetHead(Box::new(e)))?;
 
+                let reference = repository
+                    .find_reference(local_branch_ref.to_str().unwrap_or_default())
+                    .map_err(|e| Error::SetHead(Box::new(e)))?;
```

Comment:
Don't we already have the reference? Like, why are we trying to find it? Is it to make sure it exists?

Reply:
Can return from `reference_symbolic` and remove extraneous lookup.

Reply:
> Don't we already have the reference? Like, why are we trying to find it? Is it to make sure it exists?

Need to announce the `Rev::Git(oid)`

6fe4e73 removes additional lookup